app/my_api/v2/auth.py

```diff
@@ -106,12 +107,12 @@ def post(self):
                 return {'Message': 'Password is empty'}, 400
             elif not passwordconfirm:
                 return {'Message': 'Password confirm is empty'}, 400
-            elif len(password) < 8:
-                return {'Message': 'Password should have minimum of 8 characters'}, 400
+            if not re.match(r"(^[a-zA-Z0-9_.-]+@[a-zA-Z-]+\.[a-zA-Z-]+$)", email):
```

Comment:
This regex will fail to prevent an email address that starts with a digit. I.e `1jhon@gmail.com` would be valid as per this regex